samples/stock/demo/inverted-navigator/demo.js

```diff
@@ -0,0 +1,462 @@
+                enabled: false
+            }
+        }
+    },
+    title: {
+        text: 'List of stock historic inventions.',
+        align: 'left'
+    },
+    subtitle: {
+        text: 'Source: Wikipedia',
+        align: 'left'
+    },
+    rangeSelector: {
+        buttons: [{
+            type: 'year',
+            count: 1,
+            text: '1y'
+        }, {
+            type: 'year',
+            count: 30,
+            text: '20y'
+        }, {
+            type: 'year',
+            count: 100,
+            text: '100y'
+        }]
+    },
+    navigator: {
+        opposite: true,
+        series: {
+            type: 'line',
+            marker: {
+                enabled: true
+            },
+            lineWidth: 0
+        },
+        xAxis: {
+            ordinal: false,
+            reversed: false,
+            tickPixelInterval: 50
+        }
+    },
+    xAxis: {
+        ordinal: false,
+        labels: {
+            enabled: false
+        },
+        tickLength: 0,
+        lineWidth: 0,
+        min: Date.UTC(1900, 0, 1),
+        reversed: false
+    },
+    yAxis: {
+        min: 0.75,
+        max: 2,
+        labels: {
+            enabled: false
+        },
+        gridLineWidth: 0
+    },
+    tooltip: {
+        shared: false,
+        useHTML: true,
+        styles: {
+            'white-space': 'normal'
+        },
+        formatter: function () {
+            let tooltipString = '<div>';
+
+            if (this.series.name === 'Series 1') {
+                return false;
+            }
+
+            if (this.point.options.location) {
+                tooltipString = '<div style="width: 140px;">Location: ' +
+                    this.point.options.location + '</div>';
+            }
+            tooltipString += '<div style="width: 140px;">Year: ' +
+                Highcharts.dateFormat('%Y', this.x) + '</div>';
+            if (this.point.options.details) {
+                tooltipString += '<div style="width: 300px;"> ' +
+                    this.point.options.details + '</div>';
+            }
+            return tooltipString + '</div>';
+        }
+    },
+    series: [{
+        // pointStart: -10950249600000,
+        pointStart: Date.UTC(1600, 0, 1),
+        pointInterval: 5,
+        pointIntervalUnit: 'year',
+        data: [
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75
+
+        ],
+        states: {
```

Comment:
Could we also disable [the inactive state](https://api.highcharts.com/highcharts/series.line.states.inactive)? I think in this particular demo, it doesn't help